src/controls/Keyboard.ts

```diff
@@ -1,34 +1,25 @@
 import { useEffect } from 'react'
-import { cameras, useStore } from '../store'
-
-interface KeyConfig extends KeyMap {
-  keys?: string[]
-}
-
-interface KeyMap {
-  fn: (pressed: boolean) => void
-  up?: boolean
-  pressed?: boolean
-}
+import { useStore } from '../store'
+import type { KeyConfig, KeyMap } from '../store'
 
 function useKeys(keyConfig: KeyConfig[]) {
   useEffect(() => {
-    const keyMap = keyConfig.reduce<{ [key: string]: KeyMap }>((out, { keys, fn, up = true }) => {
-      keys && keys.forEach((key) => (out[key] = { fn, pressed: false, up }))
+    const codeMap = keyConfig.reduce<{ [key: string]: KeyMap }>((out, { keys, fn, up = true }) => {
```

Comment:
why did you rename `keyMap` and `key`?

Reply:
At first there was another idea of how I was going to manage keys. Returned the old name